docs/components/console/manage-clusters/manage-api-clients.md

```diff
@@ -40,6 +40,10 @@ When the rate limit is triggered, the client will receive an HTTP 429 response.
 
 ## Create a client
 
+:::caution
+With **8.7** clusters, the client scopes are now managed through the [Identity](/components/identity/manage-api-clients.md) component. The only scope still managed in Console is the cluster **Secrets** scope. To learn more about managing clients in **8.7** clusters, refer to the [Identity](/components/identity/manage-api-clients.md) documentation.
```

Comment:
The link you have here is in Console, not Identity -- what page are you trying to link to here?

Reply:
This is only a placeholder. Afaik the Identity Documentation is still not ready.